source/guides/tooling/code-coverage.md

```diff
@@ -173,6 +177,23 @@ Instead of using the `npx instrument` command, we can use {% url "`babel-plugin-
 
 We can now serve the application and get instrumented code without an intermediate folder, but the result is the same instrumented code loaded by the browser, with the same `window.__coverage__` object keeping track of the original statements.
 
+If you're using `create-react-app` serving the instrumented application is super easy: 
+
+First, install the [inst](https://github.com/cypress-io/instrument-cra) module, 
```

Comment:
`[inst]` is wrong, syntax (command followed by `Then,`).
Plus I would not say "is super easy" because it might not appear that easy to someone else.

I would not even have this paragraph, and instead, I would point people to https://github.com/cypress-io/code-coverage#examples

Reply:
I've removed the paragraph 👍🏻